src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -861,12 +861,14 @@ public void validateEndpoints(String awsRegion, String iotCredEndpoint, String i
             throw new ComponentConfigurationValidationException(
                     String.format("Error looking up AWS region %s", awsRegion), DeploymentErrorCode.UNSUPPORTED_REGION);
         }
-        if (Utils.isNotEmpty(iotCredEndpoint) && !iotCredEndpoint.contains(awsRegion)) {
+        if (Utils.isNotEmpty(iotCredEndpoint) && iotCredEndpoint.contains("amazonaws")
```

Comment:
let's make this `.amazonaws.` so that `abcamazonaws.com` doesn't trigger it.

Reply:
pull out to a const.

Reply:
lol, that's creative

Reply:
Oops missed the const comment